plaso/parsers/sqlite_plugins/ios_imohdchat_message.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for IMO HD chat message database files."""
+
+from dfdatetime import posix_time as dfdatetime_posix_time
+
+from plaso.containers import events
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+class IMOHDChatMessageEventData(events.EventData):
+  """IMO HD chat message event data.
+
+  Attributes:
+    application (str): name of the application.
+    bundle_identifier (str): bundle identifier of the application.
+    count (int): number of occurances of the event.
+    event (str): event.
+    last_used_time (dfdatetime.DateTimeValues): last date and time
+        the application was last used.
+    query (str): SQL query that was used to obtain the event data.
+  """
+
+  DATA_TYPE = 'ios:imohdchat_message:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(IMOHDChatMessageEventData, self).__init__(
+        data_type=self.DATA_TYPE)
+    self.zalias = None
+    self.ztext = None
+    self.zissent = None
+    self.zts = None
+    self.query = None
+
+
+class IMOHDChatMessagePlugin(interface.SQLitePlugin):
+  """SQLite parser plugin for IMO HD chat message database files.
+  """
+
+  NAME = 'ios_imohdchat_message'
+  DATA_FORMAT = (
+      'IMO HD chat message SQLite database (IMODb2.sqlite) file')
+
+  REQUIRED_STRUCTURE = {
+      'ZIMOCHATMSG': frozenset([
+          'Z_PK', 'ZALIAS', 'ZTEXT', 'ZISSENT',
+          'ZTS'])}
+
+  QUERIES = [(
+      ('SELECT ZTS/1000000000, ZALIAS, ZTEXT, ZISSENT '
```

Comment:
ZTS/1000000000 will remove precision, which maybe not forensically sound

ORDER BY ZTS is not needed given psort will sort events